src/react-freshchat.js

```diff
@@ -136,6 +136,15 @@ class FreshChat extends React.Component {
   componentWillUnmount() {
     widget().close()
   }
+
+  componentWillReceiveProps(nextProps) {
+    let { token, ...moreProps } = nextProps;
+    this.init({
+      host: 'https://wchat.freshchat.com',
+      token,
+      ...moreProps,
```

Comment:
It doesn't seem like `token` is used anywhere else, so why explicitly did you destructure `token` and `moreProps`, instead of just `...nextProps` here?